WebApi/WebApi/Controllers/CommentController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using WebApi.Models.DTO;
+using WebApi.Services.Services_Interfaces;
+
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController : ControllerBase
+    {
+        private readonly ICommentService _commentService;
+        private readonly ILogger<CommentController> _logger;
+        public CommentController(ILogger<CommentController> logger, ICommentService commentService)
+        {
+            _commentService = commentService;
+            _logger = logger;
+        }
+        [HttpGet]
+        public ActionResult<IQueryable<CommentDTO>> GetAll([FromHeader] int userId)
+        {
+            var result = _commentService.GetAll(userId);
+            if (result == null) return NotFound();
+            return Ok(result);
+        }
+        [HttpGet("{id}")]
+        public ActionResult<CommentDTO> GetById([FromRoute] int id, [FromHeader] int userId)
+        {
+            var result = _commentService.GetById(id, userId);
+            if (result == null) return NotFound();
+            return Ok(result);
+        }
+
+        [HttpPost]
+        public async Task<ActionResult<int>> AddComment([FromBody] CommentDTO comment, [FromHeader] int userId)
+        {
```

Comment:
[Required] przed [FromHeader]
Dlaczego przy dodawaniu zwracasz not found?
ma być bad request

Reply:
to jest głupie, ale w specyfikacji jest błąd 404, czyli NotFound()

Reply:
to zmienić na BadRequest a w specyfikacji zmienie zaraz